tools/setup_helpers/cuda.py

```diff
@@ -11,10 +11,13 @@
 
 
 def find_nvcc():
-    if IS_WINDOWS:
-        proc = Popen(['where', 'nvcc.exe'], stdout=PIPE, stderr=PIPE)
-    else:
-        proc = Popen(['which', 'nvcc'], stdout=PIPE, stderr=PIPE)
+    try:
+        if IS_WINDOWS:
+            proc = Popen(['where', 'nvcc.exe'], stdout=PIPE, stderr=PIPE)
+        else:
+            proc = Popen(['which', 'nvcc'], stdout=PIPE, stderr=PIPE)
+    except Exception:
+        return None
```

Comment:
You can consider copying out the which utility to `__init__.py`:

https://github.com/pytorch/pytorch/blob/a6441c00d663892e5b1d2fc2e07bb8ad9f4244a0/tools/setup_helpers/cmake.py#L25

Reply:
Usually, the `__init__.py` should remain empty, what about something like `common.py` or `utils.py`?

Reply:
Oh, there is `env.py`. I think I'll move it there.